src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/DocumentHelperTest.java

```diff
@@ -1678,6 +1680,10 @@ public void buildDocumentContentScot3() {
     @Test
     public void buildDocumentContentScot4() throws URISyntaxException, IOException {
         var expectedResult = getExpectedResult("expectedDocumentContentScot4.json");
+        expectedResult = expectedResult.replace("current-date", LocalDate.now().format(DateTimeFormatter
+                .ofLocalizedDate(FormatStyle.LONG)).toString());
+        expectedResult = expectedResult.replace("plus28", LocalDate.now().plusDays(28).format(DateTimeFormatter
+                .ofLocalizedDate(FormatStyle.LONG)).toString());
```

Comment:
if you check the other tests, they use 'UtilHelper.formatCurrentDate(LocalDate.now())' and 'UtilHelper.formatCurrentDatePlusDays(LocalDate.now(), 28)' rather than formatting the date and converting to string